example/transitive_imports.dart

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
```

Comment:
Should this be called `example/crawl_async.dart` so it matches the thing its an example of?

Reply:
Done